layers/state_tracker/cmd_buffer_state.h

```diff
@@ -213,6 +213,8 @@ class CMD_BUFFER_STATE : public REFCOUNTED_NODE {
         VkCullModeFlags cull_mode;
         // VK_DYNAMIC_STATE_PRIMITIVE_TOPOLOGY
         VkPrimitiveTopology primitive_topology;
+        // VK_DYNAMIC_STATE_SAMPLE_LOCATIONS_EXT
+        VkSampleLocationsInfoEXT sampleLocationsInfo;
```

Comment:
```suggestion
        VkSampleLocationsInfoEXT sample_locations_info;
```

to match the naming of others